vscode/webviews/chat/StorageWarningBanner.tsx

```diff
@@ -41,7 +41,7 @@ export const StorageWarningBanner = ({ extensionAPI, vscodeAPI }: StorageWarning
                 <header>
                     <h1 className="tw-text-blue-500">Chat History Storage is Full</h1>
```

Comment:
```suggestion
                    <h1 className="tw-text-blue-500">Large Chat History Detected</h1>
```

This is still wrong because it's not actually full?

Reply:
Good point tho